### DISCUSSION TITLE
Help with 5800x, 64gb ram, 2x2tb NVME

### DISCUSSION THREAD

**Discussion by Rogue38** [swar/Swar-Chia-Plot-Manager#1207] [Q&A]
Been trying multiple configs but cant seem to get near 30 plots a day, stuck in the low 20s right now. Been playing with timings and job amounts for a few weeks. Any advice will be greatly appreciated. 
CPU- 5800x
Ram- 64gb
NVMe - 2tb rocket Q (gen3), 2tb Sabrient Rocket (gen4)
Current settings
  max_for_phase_1: 5
  minimum_minutes_between_jobs: 5
  - name: SSD 1
    max_plots: 80
    farmer_public_key: 
    pool_public_key:
    temporary_directory: D:\
    temporary2_directory:
    destination_directory: F:\
    size: 32
    bitfield: true
    threads: 2
    buckets: 128
    memory_buffer: 4000
    max_concurrent: 3
    max_concurrent_with_start_early: 5
    initial_delay_minutes: 0
    stagger_minutes: 45
    max_for_phase_1: 1
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 15
    temporary2_destination_sync: false
    exclude_final_directory: true
    skip_full_destinations: true
    unix_process_priority: 10
    windows_process_priority: 32
    enable_cpu_affinity: false
    cpu_affinity: [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15 ]

  - name: SSD 2
    max_plots: 80
    farmer_public_key: 
    pool_public_key:
    temporary_directory: Z:\
    temporary2_directory:
    destination_directory: G:\
    size: 32
    bitfield: true
    threads: 2
    buckets: 128
    memory_buffer: 4000
    max_concurrent: 4
    max_concurrent_with_start_early: 7
    initial_delay_minutes: 0
    stagger_minutes: 45
    max_for_phase_1: 2
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 15
    temporary2_destination_sync: false
    exclude_final_directory: true
    skip_full_destinations: true
    unix_process_priority: 10
    windows_process_priority: 32
    enable_cpu_affinity: false
    cpu_affinity: [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15 ]

**Comment by w4d3j** [comment]
Are both of your drives QLC? Could just be I/O bottleneck

**Reply by Rogue38** [reply]
Then gen3 is QLC and the gen4 is TLC
